frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -39,6 +39,7 @@ type ConvertOpt struct {
 	SessionID    string
 	BuildContext *llb.State
 	Excludes     []string
+	IgnoreCache  []string
```

Comment:
Please add a comment so as to clarify that:
* slice elements are case-insensitive stage names
* nil stands for "none"
* non-nil zero-length slice stands for "all"
?